cfgen/cfgen

```diff
@@ -1275,6 +1312,46 @@ class ConfPool(ToDhall):
                           f'pvers = {oids_to_dhall(self.pvers)}'])
         return '{ %s }' % args
 
+    @classmethod
```

Comment:
Need any changes under
Failures.__doc__ = """
?

Reply:
I don't think so, documentation is pretty much generic.